share/availability/full/reconstruction_test.go

```diff
@@ -1,284 +0,0 @@
-//go:build !race
-
-package full
-
-import (
-	"context"
-	"sync"
-	"testing"
-	"time"
-
-	"github.com/stretchr/testify/require"
-	"golang.org/x/sync/errgroup"
-
-	"github.com/celestiaorg/celestia-node/header/headertest"
-	"github.com/celestiaorg/celestia-node/share"
-	"github.com/celestiaorg/celestia-node/share/availability/light"
-	availability_test "github.com/celestiaorg/celestia-node/share/availability/test"
-	"github.com/celestiaorg/celestia-node/share/eds"
-)
-
-func init() {
-	eds.RetrieveQuadrantTimeout = time.Millisecond * 100 // to speed up tests
-}
-
-// TestShareAvailable_OneFullNode asserts that a full node can ensure
-// data is available (reconstruct data square) while being connected to
-// light nodes only.
-func TestShareAvailable_OneFullNode(t *testing.T) {
```

Comment:
We should keep those tests but comment instead, so we can quickly rewrite once we get to reconstruction

We discussed that we need them and agreed to keep them on the availability level.